### ISSUE TITLE
Error while running makemigrations

### ISSUE THREAD

**Issue by bhavin192** [miniSat/minisat#32]
I'm getting following error while running `makemigrations` command.
```
django.db.utils.OperationalError: no such table: main_app_compute_resource_model
```

#### How to reproduce?
- Checkout to branch `testing`
- Delete the file `db.sqlite3`
- Run the command `python3 manage.py makemigrations`

Complete stack trace:
```
Traceback (most recent call last):
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 303, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: main_app_compute_resource_model

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/core/management/__init__.py", line 347, in execute
    django.setup()
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/apps/registry.py", line 112, in populate
    app_config.import_models()
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/bhavin192/minisat/main_app/models.py", line 38, in <module>
    class Create_host_model(models.Model):
  File "/home/bhavin192/minisat/main_app/models.py", line 40, in Create_host_model
    newlist = list(zip(newlist, newlist))
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/models/query.py", line 272, in __iter__
    self._fetch_all()
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/models/query.py", line 1179, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/models/query.py", line 179, in __iter__
    return chain.from_iterable(compiler.results_iter(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size))
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/models/sql/compiler.py", line 1014, in results_iter
    results = self.execute_sql(MULTI, chunked_fetch=chunked_fetch, chunk_size=chunk_size)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/models/sql/compiler.py", line 1063, in execute_sql
    cursor.execute(sql, params)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/bhavin192/minisat/env/lib64/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 303, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: main_app_compute_resource_model
```

**Comment by CURIOUSER005** [comment]
#36 
Removed the database file and all the files in migrations directory except __init__.py
Now you can run the command python manage.py migrate is followed by python manage.py makemigrations

There will be no error

**Comment by khomesh24** [comment]
@bhavin192 
Thanks for bringing this issue to our notice we have fixed the issue with commit 9b69a2e37eecf5b2341c29bb87b3b105750cbe95
